sdk/core/Azure.Core/tests/public/ModelReaderWriterTests/EnvelopeTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Core.Tests.Public.ModelReaderWriterTests
+{
+    internal class EnvelopeTests// : ModelJsonTests<Envelope<EnvelopeTests.ModelC>>
```

Comment:
Why include a file where the content is commented out?

Reply:
So I remember to uncomment it when we move over to supporting this case.  We hid it for first release.

Reply:
Would it make sense to file a tracking issue and add a comment regarding the intent?
Without that, it would be reasonable for someone to delete this file in a clean-up pass.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/40161